frontend/src/components/Statistics/Stats.tsx

```diff
@@ -0,0 +1,21 @@
+import { Icon } from "../Icon/Icon"
+
+type StatsProps = {
```

Comment:
Comments for the properties to unify code

Reply:
Ja comments sind ein muss. bitte adden